docs/_snippets/patient.mdx

```diff
@@ -65,3 +65,12 @@
   </Expandable>
 
 </ParamField>
+
+<ParamField query="externalId" type="string">
+  An external Patient ID to associate to a Patient in Metriport.
+</ParamField>
+
+<Info>
+  If the optional externalId param is provided you can expect to see it returned in the responses
+  involving the Patient. It will also be included within the webhook payloads.
```

Comment:
nit: comma, and highlight the param:

"If the optional `externalId` param is provided, you can expect to see it returned in the responses involving the Patient. It will also be included within the webhook payloads."